extensions/geode-modules-session/src/test/java/org/apache/geode/modules/session/internal/filter/SessionReplicationIntegrationJUnitTest.java

```diff
@@ -170,10 +171,15 @@ public void call(HttpServletRequest request, HttpServletResponse response)
 
     List<Cookie> cookies = getCookies(response);
     assertEquals("Session id != JSESSIONID from cookie", response.getContent(),
-        cookies.get(0).getValue());
+        getSessionCookie(cookies).getValue());
 
     Region r = getRegion();
-    assertNotNull("Session not found in region", r.get(cookies.get(0).getValue()));
+    assertNotNull("Session not found in region", r.get(getSessionCookie(cookies).getValue()));
+  }
+
+  private Cookie getSessionCookie(List<Cookie> cookies) {
+    return cookies.stream().filter(cookie -> cookie.getName().equals("JSESSIONID"))
```

Comment:
It looks like this method relies on the two JSESSIONID cookies being returned in a certain order.  Do you know if it safe to rely on that order?  Also curious why we are ending up with two of these cookies.

Reply:
Not sure why we have two cookies.  May be related to the servlet tester?  @upthewaterspout ^^^

Other session tests already use jetty 9.4.x (see `geode-assembly/src/test/java/org/apache/geode/session/tests/GenericAppServerInstall.java`).

Reply:
Unfortunately it's not just because of the servlet tester. It's because we set a cookie called JSESSIONID, which is the same cookie that the container is supposed to set. In the newer version if Jetty it is returning both cookies to the browser.

We really should be using a different cookie, but that may break compatibility for people already using the module.